### PR TITLE
Add a page about choosing between .com/.org

### PR DESCRIPTION
**Pull request by ericholscher**
This is an initial attempt,
but I think it’s important to document this.
We get a lot of questions so it seems obvious we need to explain it better.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -45,6 +45,7 @@ to help you create fantastic documentation for your project.
   :doc:`With Sphinx </intro/getting-started-with-sphinx>` |
   :doc:`With MkDocs </intro/getting-started-with-mkdocs>` |
   :doc:`Feature Overview </features>`
+  :doc:`Choosing between our two sites </choosing-a-site>`
```
Comment (ericholscher) [inline docs/index.rst:48]: Not sure what to call these, really. It likely needs to be more explicit.

File: docs/choosing-a-site.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
+Choosing between our two sites
+==============================
+
+A question our users often have is what the difference is between |org_brand| and |com_brand|.
+This page will lay out the functional and philosophical differences between the two sites,
+which should help you choose which is a better fit for your organization.
+
+The features available on both platforms are the same.
```
Comment (stsewd) [inline docs/choosing-a-site.rst:8]: This isn't quite true... There are more features in .com.

File: docs/choosing-a-site.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+|org_brand| is meant for open source projects to use for documentation hosting.
+This is great for user and developer documentation for your project.
+
+Important points:
+
+* All documentation sites have advertising
+* Only supports public VCS repositories
+* All documentation is publicly accessible to the world
+* Less build time and fewer build resources (memory & CPU)
```
Comment (stsewd) [inline docs/choosing-a-site.rst:22]: Maybe a footnote or clarification that users can ask to increase these limits if needed

File: docs/choosing-a-site.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+|com_brand|
+-------------------
+
+|com_brand| is meant for companies and users who have private documentation.
+It works well for product documentation as well as internal docs for your developers.
+
+Important points:
+
+* No advertising
+* Allows importing *private*and public repositories from VCS.
```
Comment (stsewd) [inline docs/choosing-a-site.rst:33]:
```suggestion
* Allows importing *private* and public repositories from VCS.
```

File: docs/choosing-a-site.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
+Choosing between our two sites
```
Comment (stsewd) [inline docs/choosing-a-site.rst:1]: We use title case for titles.

File: docs/choosing-a-site.rst (hunk 2 above)
Comment (ericholscher) [reply docs/choosing-a-site.rst:8]: Like what?

File: docs/choosing-a-site.rst (hunk 3 above)
Comment (ericholscher) [reply docs/choosing-a-site.rst:22]: The goal is selling the .com ;)

File: docs/choosing-a-site.rst (hunk 2 above)
Comment (stsewd) [reply docs/choosing-a-site.rst:8]: Organizations, and sharing

File: docs/choosing-a-site.rst (hunk 3 above)
Comment (stsewd) [reply docs/choosing-a-site.rst:22]: I had a user who wanted to change its open source project to .com bc of more resources

File: docs/choosing-a-site.rst (hunk 2 above)
Comment (ericholscher) [reply docs/choosing-a-site.rst:8]: Yea, those are mentioned here. They aren't really functional differences.

File: docs/index.rst [hunk 6]
```diff
@@ -45,6 +45,7 @@ to help you create fantastic documentation for your project.
   :doc:`With Sphinx </intro/getting-started-with-sphinx>` |
   :doc:`With MkDocs </intro/getting-started-with-mkdocs>` |
   :doc:`Feature Overview </features>`
```
Comment (davidfischer) [inline docs/index.rst:47]:
```suggestion
  :doc:`Feature Overview </features>` |
```

**Comment by ericholscher** [comment]
Going to ship this, since it's definitely something users have had questions about.

File: docs/choosing-a-site.rst [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+Important points:
+
+* All documentation sites have advertising
+* Only supports public VCS repositories
+* All documentation is publicly accessible to the world
+* Less build time and fewer build resources (memory & CPU)
+* Documentation is organized by projects
+
+You can sign up for an account at https://readthedocs.org.
```
Comment (humitos) [inline docs/choosing-a-site.rst:25]: What about adding "free account" here?

File: docs/choosing-a-site.rst [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+Important points:
+
+* No advertising
+* Allows importing private and public repositories from VCS
+* Supports private versions that only your organization or people you give access to can see
+* More build time and more build resources (memory & CPU)
+* Documentation is organized by organization, giving more control over permissions
+
+You can sign up for an account at https://readthedocs.com.
```
Comment (humitos) [inline docs/choosing-a-site.rst:41]: Maybe mentioning "you can sign up for a 30-days free trial"?

File: docs/choosing-a-site.rst [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+* Less build time and fewer build resources (memory & CPU)
+* Documentation is organized by projects
+
+You can sign up for an account at https://readthedocs.org.
+
+Read the Docs for Business
+--------------------------
+
+|com_brand| is meant for companies and users who have private documentation.
+It works well for product documentation as well as internal docs for your developers.
```
Comment (humitos) [inline docs/choosing-a-site.rst:31]: Instead of "developers", maybe "employees"?

**Pull request merged**